src/system-health/scripts/healthd

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python2
+
+"""
+    healthd
+    System health monitor daemon for SONiC
+"""
+
+import signal
+import threading
+from sonic_daemon_base.daemon_base import Logger
+from sonic_daemon_base.daemon_base import DaemonBase
```

Comment:
sonic_daemon_base is being deprecated in favor of the new sonic-py_common package. Please update accordingly. See https://github.com/Azure/sonic-buildimage/pull/5043 and https://github.com/Azure/sonic-platform-daemons/pull/74 for reference.

Reply:
Sure